packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -156,6 +156,7 @@
 	"aiAssistant.newSessionModal.question": "Are you sure you want to start a new session?",
 	"aiAssistant.newSessionModal.confirm": "Start new session",
 	"aiAssistant.serviceError.message": "Unable to connect to n8n's AI service ({message})",
+	"aiAssistant.payloadTooBig.message": "Payload size is too big, try removing unnecessary data from the workflow and try again.",
```

Comment:
🤔 Not sure how I would go about doing that as a user. Maybe only removing nodes or trying with a smaller workflow. 
Maybe let's leave it as
```
Cannot reach service because payload size is too big. 
```